wagtail/admin/templates/wagtailadmin/home/upgrade_notification.html

```diff
@@ -1,17 +1,20 @@
 {% load i18n wagtailcore_tags wagtailadmin_tags %}
 {% wagtail_version as current_version %}
-<div
-    class="w-panel-upgrade panel w-hidden"
-    data-controller="w-upgrade"
-    data-w-upgrade-current-version-value="{{ current_version }}"
-    {% if lts_only %}data-w-upgrade-lts-only-value="true"{% endif %}
-    data-w-upgrade-hidden-class="w-hidden"
->
-    <div class="help-block help-warning">
-        {% icon name='warning' %}
-        {% blocktrans trimmed %}
-            Wagtail upgrade available. Your version: <strong>{{ current_version }}</strong>. New version: <strong data-w-upgrade-target="latestVersion"></strong>.
-        {% endblocktrans %}
-        <a href="" data-w-upgrade-target="link">{% trans "Read the release notes." %}</a>
+<div class="w-panel-upgrade w-hidden w-flex w-gap-5 w-items-center w-pl-slim-header w-pr-5 sm:w-px-[3.5rem] w-py-5 w-color-text-content w-bg-surface-info-panel"
```

Comment:
This would also need a bottom border for forced colors mode, like we do for success / warning / error messages.